packages/@okta/vuepress-site/docs/reference/rl-best-practices/index.md

```diff
@@ -194,22 +194,25 @@ The error condition resolves itself as soon as there is another concurrent threa
 
 ## Review your API limit parameter
 
-In some cases, you can avoid hitting rate limits by implementing the maximum value of the `limit` query parameter for an individual API endpoint. Increasing this value to the maximum reduces the number of calls for a given operation, which can keep you under the endpoint's rate limit. By default, Okta uses a default `limit` if one isn't set.
+Sometimes, you can avoid hitting rate limits by implementing the maximum value of the `limit` query parameter for an individual API endpoint. Increasing this value to the maximum reduces the number of calls for a given operation, which can keep you under the endpoint's rate limit. By default, Okta uses a default `limit` if one isn't set.
 
 For example, the endpoint `/api/v1/apps/{applicationId}/users` returns, by default, 50 results. You can increase this limit to 500, reducing the number of calls.
 
 See the [Core Okta API](/docs/reference/core-okta-api/) for details on each endpoint's default and maximum values for the `limit` parameter.
 
 ## Request rate limit exceptions
 
-You can adjust the size and structure of rate limits by requesting a temporary rate limit increase. For example, if you are importing a large number of users and groups, you may need a temporary rate limit increase. See [How to Request a Temporary Rate Limit Increase](https://support.okta.com/help/s/article/How-can-we-request-to-have-the-rate-limit-for-our-org-temporarily-increased?language=en_US).
+You can submit a request by opening a support case to increase the rate limit for specific Okta API endpoints. These requests must be submitted at least 15 business days before the increased rate limit is needed. After submission, Okta Support teams review each request thoroughly and either grant or deny the increase. Okta strongly recommends submitting requests as far in advance as possible. Some requests may take longer to review.
 
-To request an exception, contact [Support](https://support.okta.com/help/open_case) a minimum of 10 days before you need the increase and provide the following details:
+Requests can be submitted for the following customers:
```

Comment:
```suggestion
You can submit requests for the following customers:
```

Reply:
How about "The following customers can submit requests for rate limit exceptions:"